ci-operator/jobs/openshift/odo/openshift-odo-master-periodics.yaml

```diff
@@ -287,3 +287,81 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 */6 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: odo
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: v4.6
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-odo-master-v4.6-integration-e2e-periodic
```

Comment:
Prow job generator does not carry the notification config. First you need to add it manually then run `make jobs` to confirm the change.  
```
reporter_config:
    slack:
      channel: '#odo-notifications'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow: {{else}} :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :volcano: {{end}}'
```

For reference check - https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/odo/openshift-odo-master-periodics.yaml#L17-L24

Reply:
Ahh... missed it thanks for pointing it out 👍

Reply:
done